pulsar-broker/src/test/java/org/apache/pulsar/broker/service/InactiveTopicDeleteTest.java

```diff
@@ -324,20 +324,33 @@ public void testDeleteWhenNoBacklogs() throws Exception {
             .subscriptionName("sub")
             .subscribe();
 
-        for (int i = 0; i < 10; i++) {
+        Consumer<byte[]> consumer2 = pulsarClient.newConsumer()
+                .topicsPattern("persistent://prop/ns-abc/test.*")
```

Comment:
Question:
Shall we create multiple topics to test?

Reply:
added.